server/aws-lsp-identity/src/language-server/profiles/profileService.ts

```diff
@@ -0,0 +1,243 @@
+import {
+    ListProfilesError,
+    ListProfilesParams,
+    ListProfilesResult,
+    Profile,
+    ProfileKind,
+    SsoSession,
+    UpdateProfileError,
+    UpdateProfileErrorData,
+    updateProfileOptionsDefaults,
+    UpdateProfileParams,
+    UpdateProfileResult,
+    AwsErrorCodes,
+    AwsResponseError,
+    AwsResponseErrorData,
+} from '@aws/language-server-runtimes/server-interface/identity-management'
+import { CancellationToken } from 'vscode-languageserver'
```

Comment:
nit: `server-interface` re-exports `CancellationToken`, so `vscode-languageserver` dependency can be omitted

Reply:
I'll update this in the upcoming PR to avoid further delay on this one.